src/styles/global.scss

```diff
@@ -140,6 +140,7 @@ h1 {
 .robot-button {
   background: none;
   border: 0;
+  height: min(30vw, 235px);
```

Comment:
```suggestion
```
Can we remove this? It's messing up the layout. Unless there's some reason to have it, let me know. Looks good otherwise!

Reply:
Before I added the Reward component the robots were pretty large when the page was viewed at full height and they could shrink a little bit as the screen size decreased. Once I added the Reward component around them the robots stayed at the minimum size no matter what, so this makes them back to the max height again. They do not change size when the screen shrinks though since I didn't know how to do that.

Reply:
Okay, I see. I'm not sure that's the correct way to fix this issue so let's merge this PR without that line and I'll take a look at the sizing when I do my CSS bug run-through this week.